lua/mateditor/sh_mateditor.lua

```diff
@@ -201,14 +201,35 @@ end
 local requestQueueBatchSize = 50
 
 if CLIENT then
-	net.Receive( "AdvMatMaterialize", function()
+	local function readDecimal()
+		return net.ReadUInt( 10 ) / 100
```

Comment:
```suggestion
		return net.ReadUInt( 16 ) / 100
```

max 10 on these is pretty limiting, i personally have gone up to 32 on some builds
655 should never really limit anyone